### DISCUSSION TITLE
我们的默认主题是不是不带 date tag 此类的 Frontmatter？

### DISCUSSION THREAD

**Discussion by DrAugus** [vuepress/core#919] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/34206897/171104134-1d107b93-1834-4724-a543-ed98eb3747cf.png)
我在文档开头加入了这些，实际页面开头并没有显示，是否还需要修改其他配置呢？

**Comment by Mister-Hope** [comment, marked as answer]
`date` is a preserved field in VuePress, and it's mentioned in documatation.

Our default theme is targeting simple documatation, so it doesn't have much features.

We provide theme and plugin apis, and encourage people to develop and use theme and plugins,  so if you want more features, you should use community themes.

I maintained a theme called `vuepress-theme-hope` and is supporting the feature you ask, you can have a look by visiting: https://vuepress-theme-hope.github.io/v2/

**Reply by DrAugus** [reply]
多谢，感激不尽
